### ISSUE TITLE
Display user's location in map

### ISSUE THREAD

**Issue by newluck77** [newluck77/fprsmap#10]
Request location from browser and display a marker there + center map / zoom in.

**Comment by Roboe** [comment]
Doesn't Leaflet provide some method for this already?

**Comment by newluck77** [comment]
Should we implement this, or do you think that we'd offset the privacy-aware part of the community? ;)

**Comment by Roboe** [comment]
Yes, privacy is a thing. I've read somewhere that Google is involved somewhat in the Firefox geolocation API… need to investigate further the drawbacks.

The poll is a good idea, but once we have the above info, :ok:

**Comment by StefanBrand** [comment]
I for one never pass my location on to websites, but I'm sure there are people out there, who'd find it very convenient.

I'm still for the poll, as soon as the map has made it to the new Fairphone website ( https://fairphone.com ).

**Comment by Roboe** [comment]
Yes, I agree for the poll. Convenient browsers let you decide to pass it or not, for privacy-concerned people (they will use one of these browsers for sure)

**Comment by StefanBrand** [comment]
>Doesn't Leaflet provide some method for this already?

There is even a tutorial now: http://leafletjs.com/examples/mobile/
